docs/resources/hotels.md

```diff
@@ -22,9 +22,93 @@ Let's learn how to get started and help your users book the perfect rooms at ove
 
 ### Get a list of hotels by location 
 
-First, users should be able to search hotels for a given location. The [Hotel List API](https://developers.amadeus.com/self-service/category/hotel/api-doc/hotel-list/api-reference) returns the list of hotels based on a city or a geographic code. To answer a question, such as **"what are the hotels closed to the city hall?"** the `Hotel List API` has three endpoints to utilize based on your search criteria. It returns `hotel name`, `location`, and `hotel id` for you to proceed to the next steps of the hotel search. 
+First, users should be able to search hotels for a given location. The [Hotel List API](https://developers.amadeus.com/self-service/category/hotel/api-doc/hotel-list/api-reference) returns a list of hotels based on a city, a geographic code or the unique Amadeus hotel Id. To answer a question, such as **"what are the hotels closed to the city hall?"** the `Hotel List API` has three endpoints to utilize based on your search criteria. It returns `hotel name`, `location`, and `hotel id` for you to proceed to the next steps of the hotel search. 
 
-Based on the search criteria, you will get the list of `hotelId` with hotel information as in the example below.
+The [Hotel List API](https://developers.amadeus.com/self-service/category/hotel/api-doc/hotel-list/api-reference) contains the following endpoints:
+
+* `GET ​/reference-data​/locations​/hotels​/by-city` - searches hotels by a city code
+* `GET ​/reference-data​/locations​/hotels​/by-geocode` - searches hotels by geographic coordinates
+* `GET /reference-data​/locations​/hotels​/by-hotels` - searches hotels by a unique Amadeus hotel Id
+
+
+#### Search hotels by a city  
+
+You can specify an [IATA city code](https://www.iata.org/en/publications/directories/code-search/) or Geocode to search a more specific area to get the list of hotels. You can customize the request using parameters, such as radius, chain code, amenities, star ratings, and hotel source. 
+
+To search a hotel by a city code, the IATA city code is the only required query parameter:
+
+```bash
+GET https://test.api.amadeus.com/v1/reference-data/locations/hotels/by-city?cityCode=PAR&amenities=&ratings=
```

Comment:
Here since amenities has no values when I call the API I see:

https://test.api.amadeus.com/v1/reference-data/locations/hotels/by-city?cityCode=PAR&ratings=